packages/ndla-ui/src/Gloss/index.tsx

```diff
@@ -0,0 +1,9 @@
+/**
+ * Copyright (c) 2023-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+export { default as WordList } from './Gloss';
```

Comment:
Antar denne må være slik
```suggestion
export { default as Gloss } from './Gloss';
```